src/applications/mhv/landing-page/utilities/index.js

```diff
@@ -0,0 +1,11 @@
+function getDestinationDomain(href) {
+  if (href.includes('VAgov')) {
+    return 'VAgov';
+  }
+  if (href.includes('NationalPortal')) {
```

Comment:
I might be overlooking something, but when do the exact strings `'VAgov'` or `'NationalPortal'` appear in URLs?

Reply:
This has been fixed, that was more of a typo before.